### PR TITLE
suggested doc updates

### PR DESCRIPTION
**Pull request by JoeOster**
This PR is to start a conversation around improvements to current documents

### CODE REVIEW

File: docs/admin/administration.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ metrics from the local host only.
 `daos_metrics` displays the metrics in a human-readable tree format or CSV
 format (`--csv`).
 
-Each DAOS engine maintains its own metrics.
+Each DAOS engine maintains its metrics.
```
Comment (Michael-Hennecke) [inline docs/admin/administration.md:194]: keeping original - please revert

File: docs/admin/administration.md [hunk 2]
```diff
@@ -271,7 +271,7 @@ then the values supplied to `dmg pool create` should be
 a maximum of the SCM/NVMe free space divided by the number of ranks per host.
 
 For example, if 2.0 TB SCM and 10.0 TB NVMe free space is reported by
-`dmg storage query usage` and the server configuration file used to start the
+`dmg storage query usage` and the server configuration file used to start the
```
Comment (Michael-Hennecke) [inline docs/admin/administration.md:274]: spurious blank

File: docs/admin/administration.md [hunk 3]
```diff
@@ -418,13 +418,13 @@ Usage:
...
 The NVMe storage query device-health and target-health commands query the device
-health data, including NVMe SSD health stats and in-memory I/O error and checksum
+health data, including NVMe SSD health stats, in-memory I/O error and checksum
 error counters. The server rank and device state are also listed. The device health
 data can either be queried by device UUID (device-health command) or by VOS target ID
 along with the server rank (target-health command). The same device health information
 is displayed with both command options. Additionally, vendor-specific SMART stats are
 displayed, currently for Intel devices only. Note: A reasonable timed workload > 60 min
-must be ran for the SMART stats to register (Raw values are 65535). Media wear percentage
+must be running for the SMART stats to register (Raw values are 65535). Media wear percentage
```
Comment (Michael-Hennecke) [inline docs/admin/administration.md:427]: keeping original - please revert

File: docs/admin/administration.md [hunk 4]
```diff
@@ -569,13 +569,13 @@ boro-11
 ```
 The FAULTY device will transition from an "EVICTED" state back to a "NORMAL" state,
 and will again be available for use with DAOS. The use case of this command will mainly
-be for testing or for accidental device eviction.
+be for testing or accidental device eviction.
```
Comment (Michael-Hennecke) [inline docs/admin/administration.md:572]: or to recover from accidental

File: docs/admin/administration.md [hunk 5]
```diff
@@ -622,14 +622,14 @@ that join the DAOS system. Once an engine has joined the DAOS system, it is
...
 
-A DAOS system can be shutdown and restarted to perform maintenance and/or
-reboot hosts. Pool data and state will be maintained providing no changes are
+A DAOS system can be shut down and restarted to perform maintenance and/or
+reboot hosts. Pool data and state will be maintained, providing that no changes are
 made to the rank's metadata stored on persistent memory.
 
-Storage reformat can also be performed after system shutdown. Pools will be
-removed and storage wiped.
+Storage reformats can also be performed after a system shutdown. Pools
```
Comment (Michael-Hennecke) [inline docs/admin/administration.md:629]: keeping original - please revert
("storage reformat" is the dmg command, no plural)

File: docs/admin/administration.md [hunk 6]
```diff
@@ -709,31 +709,31 @@ Usage:
...
+The output table will indicate the action and result.
 
 While the engines are stopped, the DAOS servers will continue to
 operate and listen on the management network.
 
 !!! warning
     All engines monitor each other and pro-actively exclude unresponsive
-    members. It is critical to properly stop a DAOS system as with dmg in
-    the case of a planned maintenance on all or a majority of the DAOS
+    members. It is critical to stop a DAOS system with dmg in properly in
```
Comment (Michael-Hennecke) [inline docs/admin/administration.md:722]: It is critical to properly stop a DAOS system with the `dmg system stop` command at the beginning of a planned ...

File: docs/admin/administration.md [hunk 7]
```diff
@@ -709,31 +709,31 @@ Usage:
...
+    the case of planned maintenance on all or a majority of the DAOS
     storage nodes. An abrupt reboot of the storage nodes might result
-    in massive exclusion that will take time to recover.
+    In massive exclusion, that will take time to recover.
```
Comment (Michael-Hennecke) [inline docs/admin/administration.md:725]: keeping original - please revert

File: docs/admin/administration.md [hunk 8]
```diff
@@ -811,7 +811,7 @@ formatted again by running `dmg storage format`.
     The `/dev/pmemX` devices will remain mounted,
     and the PMem configuration will not be reset to Memory Mode.
     To completely unconfigure the SCM, it is advisable to run
-    `daos_server storage prepare --scm-only --reset` which will
+    `daos_server storage prepare --SCM-only --reset` which will
```
Comment (Michael-Hennecke) [inline docs/admin/administration.md:814]: keeping original - please revert

File: docs/admin/administration.md [hunk 9]
```diff
@@ -821,20 +821,20 @@ formatted again by running `dmg storage format`.
...
 
 - the relevant certificates
-- the server yaml file pointing to the access points of the running
+- the server yml file pointing to the access points of the running
   DAOS system
 
 The daos\_control.yml file should also be updated to include the new DAOS server.
 
-Then starts the daos\_server via systemd and format the new server via
+Then start the daos\_server via the systemd command and format the new server via
```
Comment (Michael-Hennecke) [inline docs/admin/administration.md:829]: keeping original - please revert
(systemd is the facility ... the command would be systemctl)

**Review by Michael-Hennecke — CHANGES_REQUESTED** [review]
Thanks Joe for the thorough review. I have commented inline on a few of your edits that I believe should not be applied - can you revert those?

File: docs/admin/administration.md (hunk 9 above)
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md (hunk 7 above)
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md (hunk 8 above)
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md (hunk 6 above)
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md (hunk 5 above)
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md (hunk 4 above)
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md (hunk 3 above)
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md (hunk 2 above)
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md (hunk 1 above)
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md [hunk 10]
```diff
@@ -44,39 +44,38 @@ severity, message, description, and cause.
...
-Settings will be applied to all running DAOS I/O Engines present in the configured
+Engine log levels can be changed dynamically (at runtime) by setting log
+Masks for a set of facilities to a given level.
+The changed settings will be applied to all running DAOS I/O Engines present in the configured
 dmg hostlist using the command `dmg server set-logmasks [<masks>]`.
 The command accepts 0-1 positional arguments.
-If no args are passed, then the log masks for each running engine will be reset
-to the value of engine "log\_mask" parameter in the server config file (as set
+If no args are passed, the log masks for each running engine will be reset
+to the value of engine "log\_mask" parameter in the server config file (asset
```
[the remark this answers is not in the document]
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md [hunk 11]
```diff
@@ -44,39 +44,38 @@ severity, message, description, and cause.
...
 parameters in the server config file. Logging is described in detail in the
-[`Debugging System`](https://daos-stack.github.io/admin/troubleshooting/#debugging-system)
-section.
+[`Debugging System`](https://daos-stack.github.io/admin/troubleshooting/#debugging-system) section.
 
-Engine log levels can be changed dynamically (at runtime) by setting log masks
-for a set of facilities to a given level.
-Settings will be applied to all running DAOS I/O Engines present in the configured
+Engine log levels can be changed dynamically (at runtime) by setting log
+Masks for a set of facilities to a given level.
```
[the remark this answers is not in the document]
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md [hunk 12]
```diff
@@ -44,39 +44,38 @@ severity, message, description, and cause.
...
+| container\_durable\_format\_incompat| INFO\_ONLY| ERROR| incompatible layout version[: <current\> not in [<min\>, <max\>\]| Indicates the given container's layout version does not match any of the versions supported by the currently running DAOS software.| DAOS engine starts with container data in local storage with s an incompatible layout version.|
+| rdb\_durable\_format\_incompatible| INFO\_ONLY| ERROR| incompatible layout version[: <current\> not in [<min\>, <max\>]] OR incompatible DB UUID: <uuid\> | Indicates the given RDB's layout version does not match any of the versions supported by the currently running DAOS software, or the given RDB's UUID does not match the expected UUID (usually because the RDB belongs to a pool created by a pre-2.0 DAOS version).| DAOS engine starts with RDB data in local storage with s an incompatible layout version.|
```
[the remark this answers is not in the document]
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md [hunk 13]
```diff
@@ -44,39 +44,38 @@ severity, message, description, and cause.
...
+| pool\_replicas\_updated| STATE\_CHANGE| NOTICE| List of pool service replica ranks has been updated.| Indicates a pool service replica list has changed. The event contains the new service replica list in a custom payload. | When a pool service replica rank becomes unavailable, a new rank is selected to replace it (if available). |
+| pool\_durable\_format\_incompat| INFO\_ONLY| ERROR| incompatible layout version: <current\> not in [<min\>, <max\>]| Indicates the given pool's layout version does not match any of the versions supported by the currently running DAOS software.| DAOS engine starts with pool data in local storage with s an incompatible layout version. |
+| container\_durable\_format\_incompat| INFO\_ONLY| ERROR| incompatible layout version[: <current\> not in [<min\>, <max\>\]| Indicates the given container's layout version does not match any of the versions supported by the currently running DAOS software.| DAOS engine starts with container data in local storage with s an incompatible layout version.|
```
[the remark this answers is not in the document]
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md [hunk 14]
```diff
@@ -44,39 +44,38 @@ severity, message, description, and cause.
...
-| rdb\_durable\_format\_incompatible| INFO\_ONLY| ERROR| incompatible layout version[: <current\> not in [<min\>, <max\>]] OR incompatible DB UUID: <uuid\> | Indicates the given RDB's layout version does not match any of the versions supported by the currently running DAOS software, or the given RDB's UUID does not match the expected UUID (usually because the RDB belongs to a pool created by a pre-2.0 DAOS version).| DAOS engine is started with rdb data in local storage that has an incompatible layout version.|
+| pool\_replicas\_updated| STATE\_CHANGE| NOTICE| List of pool service replica ranks has been updated.| Indicates a pool service replica list has changed. The event contains the new service replica list in a custom payload. | When a pool service replica rank becomes unavailable, a new rank is selected to replace it (if available). |
+| pool\_durable\_format\_incompat| INFO\_ONLY| ERROR| incompatible layout version: <current\> not in [<min\>, <max\>]| Indicates the given pool's layout version does not match any of the versions supported by the currently running DAOS software.| DAOS engine starts with pool data in local storage with s an incompatible layout version. |
```
[the remark this answers is not in the document]
Comment (JoeOster) [reply docs/admin/administration.md]: fixed

File: docs/admin/administration.md [hunk 15]
```diff
@@ -2,7 +2,7 @@
 
 ## RAS Events
 
-Reliability, Availability, and Serviceability (RAS) related events are
+Reliability, Availability, and Serviceability (RAS) related events are
```
[the remark this answers is not in the document]
Comment (JoeOster) [reply docs/admin/administration.md:5]: fixed

**Comment by JoeOster** [comment]
I have updated the document per your request

**Comment by Michael-Hennecke** [comment]
Signed-off-by: is missing from commits?

**Comment by JoeOster** [comment]
This has gotten messy somehow, I will resubmit

**Pull request closed without merging**